### ISSUE TITLE
macOS app bundle has maestral-cocoa library mismatch

### ISSUE THREAD

**Issue by ggascoigne** [samschott/maestral#672]
**Describe the bug**

I just installed Maestral using brew, it installed https://github.com/SamSchott/maestral/releases/download/v1.6.2/Maestral-1.6.2.dmg.

When I try to run the gui I get:

```sh
$ maestral gui
! maestral-cocoa>=1.6.2 required but you have 1.6.0.dev2
```

**System:**

 - Maestral version: 1.6.2
 - Python version: 3.9.12
 - OS: [e.g. Ubuntu] OSX 12.3.1

Perhaps not relevant for this, but this is running on an Apple M1 Max, macbook pro.

**Comment by samschott** [comment]
I'm afraid it's not just the homebrew version, it's the macOS app bundle in general. It does come with the correct code of version 1.6.2, but due to a bug in the packaging tool [briefcase](https://github.com/beeware/briefcare), the metadata of previous versions is included alongside the new one. It just so happens that the version check when running `maestral gui` picks up the old metadata first.

I've submitted a fix upstream at https://github.com/beeware/briefcase/pull/721. This will be fixed in Maestral's next release.

For now, you will need to start the GUI in the old-fashioned way outside of the terminal, for example by double-clicking the app in Finder.

**Comment by samschott** [comment]
Should be fixed now with v1.6.3.
